astropy/modeling/tests/test_fitters.py

```diff
@@ -24,12 +23,33 @@
 from .utils import ignore_non_integer_warning
 from ...stats import sigma_clip
 
+from ...utils.exceptions import AstropyUserWarning
+from ..fitting import populate_entry_points
+import warnings
+
+from pkg_resources import EntryPoint
```

Comment:
Did you mean to have `from pkg_resources import EntryPoint` here?
You have this line again below in a try ... except block.